.github/workflows/test.yml

```diff
@@ -150,9 +150,11 @@ jobs:
         run: |
           make coverage-check
 
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+      - name: Publish coverage report to Code Climate
+        uses: paambaati/codeclimate-action@v5.0.0
```

Comment:
I am always a bit nervous using an action from a user. How reliable is that and what is the risk of having this dependency. Is there something that checks the actions we are using?